templates/profile.php

```diff
@@ -360,7 +360,7 @@ function round(num, places) {
     <br />
     <label style='width:<?= $label_width ?>;'>Gender:</label> <?= $player->gender ?><br />
     <label style='width:<?= $label_width ?>;'>Village:</label> <?= $player->village ?><br />
-    <label style='width:<?= $label_width ?>;'>Location:</label> <?= $player->location ?><br />
+    <label style='width:<?= $label_width ?>;'>Location:</label> <?= $player->location->fetchString() ?><br />
```

Comment:
Seems we'd want to give players a more descriptive representation of what map they're on. Maybe we do just X.Y for now and later add in map name? Or map name if it's not the default